src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamNetworkStreamTest.cs

```diff
@@ -223,14 +223,7 @@ public async Task SslStream_TargetHostName_Succeeds(bool useEmptyName)
                     (sender, certificate, chain, sslPolicyErrors) =>
                     {
                         SslStream stream = (SslStream)sender;
-                        if (useEmptyName)
-                        {
-                            Assert.Equal('?', stream.TargetHostName[0]);
-                        }
-                        else
-                        {
-                            Assert.Equal(tagetName, stream.TargetHostName);
-                        }
+                        Assert.Equal(tagetName, stream.TargetHostName);
```

Comment:
Your variable `tagetName` has a Boston accent.

Reply:
updated to be geographically neutral  :)